### PR TITLE
{math}[intel/2016b] ELPA v2016.05.004

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2571.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2111da8d6597331db16cfea92e33d426 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node825.raichu.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/023037defd13eb4fc5df695f9a609448 for a full test report.

**Comment by boegel** [comment]
@damianam please review?

File: easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+version = '2016.05.004'
+
+homepage = 'http://elpa.rzg.mpg.de'
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
+sources = ['elpa-%(version)s.tar.gz']
```
Comment (damianam) [inline easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb]: Any reason to don't use `SOURCELOWER_TAR_GZ`?

File: easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+builddependencies = [('Autotools', '20150215')]
+
+configopts = '--with-generic-simple --disable-shared '
+configopts += 'FCFLAGS="-I$EBROOTIMKL/mkl/include/intel64/lp64 $FCFLAGS" LIBS="$LIBSCALAPACK"'
+buildopts = ' V=1 LIBS="$LIBSCALAPACK"'
+
+parallel = 1
```
Comment (damianam) [inline easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb]: It has built fine for me before without limiting this

File: easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
+sources = ['elpa-%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20150215')]
+
+configopts = '--with-generic-simple --disable-shared '
```
Comment (damianam) [inline easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb]: Is there a reason to force `--disable-shared`?

File: easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb]: No, will fix, thanks.

File: easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb]: OK, I'll drop it and test it on a 20-core system.

File: easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb (hunk 3 above)
Comment (boegel) [inline easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb]: This was how the ELPA easyconfig was originally contributed. I can try dropping that, and see what we end up with.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2623.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/98b95be380d666f268da5da4ae4f6045 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node801.raichu.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ff4fc353c06f9b5a7058297f637d1f01 for a full test report.

**Comment by boegel** [comment]
Remarks fixed, thanks for the review @damianam!

**Pull request merged**